internal/commands/util/usercount/github.go

```diff
@@ -161,20 +186,29 @@ func collectFromOrgs(gitHubWrapper wrappers.GitHubWrapper) ([]wrappers.CommitRoo
 					UniqueContributors: uniqueContributors,
 				},
 			)
+			for name := range uniqueContributorsName {
+				viewsUsers = append(
+					viewsUsers,
+					UserView{
+						Name:                       repository.FullName,
+						UniqueContributorsUsername: name,
+					},
+				)
+			}
 		}
 	}
-	return totalCommits, views, nil
+	return totalCommits, views, viewsUsers, nil
 }
 
-func getUniqueContributors(commits []wrappers.CommitRoot) uint64 {
+func getUniqueContributors(commits []wrappers.CommitRoot) (contributorSize uint64, contributorsList map[string]bool) {
```

Comment:
Couple of suggestions:

- named return parameters are used to only use "return" in the function, so one would define the named parameters in the body and return automatically returns those vars, doing both seems unnecessary
- list in a map name is misleading
- as we now need it, consider returning only the map and the caller can get the size if he needs, avoids the ignored return above

Reply:
Thanks, changes applied.